pkg/yang/camelcase_test.go

```diff
@@ -24,15 +24,19 @@ func TestCamelCase(t *testing.T) {
 	}{
 		{"one", "One"},
```

Comment:
related to above comment, do we want to include tests for handling invalid inputs (e.g., invalid characters like `:`) ?

Reply:
Done